AzureCalling/Controllers/CallViewController.swift

```diff
@@ -322,6 +322,7 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
 
         infoHeaderView.toggleDisplay()
         meetingInfoViewUpdate()
+        callingContext.updateScreenSharingParticipant()
```

Comment:
Does this work for joining a teams' meeting with screensharing by a link?
I don't think it's a good idea to call the seekScreenSharingParticipant from an external view controller. 
We'd better do the seeking when the call state changed to .connected

Reply:
Good call, I have move it to state change .connected.

And I tested the code locally on teams_interop branch and it works for Teams meeting screen share.